test/unit/ubuntu_test.rb

```diff
@@ -0,0 +1,45 @@
+require 'test_helper'
+
+class UbuntuTest < ActiveSupport::TestCase
+
+  test "os label for ubuntu" do
+    assert_equal "Ubuntu 14.10", operatingsystems(:ubuntu14_10).to_label
+  end
+
+  test "kernel location for 64bit arch ubuntu" do
+    host = FactoryGirl.create(:host, :operatingsystem => operatingsystems(:ubuntu14_10),
```

Comment:
These three ```host``` creation calls can be just ```.build```, which won't hit the database and make your tests faster. Same for ```debian_test``` (and ```coreos_test```, although I understand that's not part of this commit). 

Also you could put these methods in a context with a setup block, like

```ruby
context "some name" do 
  setup do 
   host = FactoryGirl.build(:host, :operatingsystem => operatingsystems(:ubuntu14_10),
     :architecture => architectures(:x86_64))
  end
 
  test ...
end
```